csaf_2.0/test/cpe/run_tests.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
```

Comment:
In my own daily work, I tend to have a `# synopsis like sigle line comment always on line 2 ... should we do this here too?

Reply:
Feel free to do so.